### ISSUE TITLE
Unclear sentence in the Comparison to alternatives page

### ISSUE THREAD

**Issue by ferdi05** [meilisearch/documentation#1102]
In the `https://docs.meilisearch.com/learn/what_is_meilisearch/comparison_to_alternatives.html#lucene` paragraph, the following sentence is unclear:

> Since Lucene has been developed to build web search applications such as Google, you can see that DuckDuckGo still uses it for some specific searches.

I'm not getting the "you can see" part

**Comment by guimachiavelli** [comment]
Agreed, it is really unclear what the original author meant with `you can see`.  I'm not fond of the overall structure of this sentence, to be honest, and would be happy to see it rewritten.

Some possible fixes:

- Remove `you can see`;
- Rewrite the phrase so it does not use `Since` and `you can see`.
